source/common/http/null_route_impl.h

```diff
@@ -103,6 +103,9 @@ struct RouteEntryImpl : public Router::RouteEntry {
 
   // Router::RouteEntry
   const std::string& clusterName() const override { return cluster_name_; }
+  const std::string getRequestHostValue(const Http::RequestHeaderMap& headers) const override {
+    return std::string(headers.getHostValue());
```

Comment:
not sure if it would be better to just return an empty string here. wtdy?

Reply:
I think this preserves prior behavior for async requests so is more correct.